src/tabnineChatWidget/ChatApi.ts

```diff
@@ -94,162 +90,167 @@ type WorkspaceFolders = {
 const CHAT_CONVERSATIONS_KEY = "CHAT_CONVERSATIONS";
 const CHAT_SETTINGS_KEY = "CHAT_SETTINGS";
 
-export function initChatApi(
-  context: vscode.ExtensionContext,
-  onInit: () => void,
-  serverUrl?: string
-) {
-  if (process.env.IS_EVAL_MODE === "true") {
-    context.subscriptions.push(
-      vscode.commands.registerCommand(
-        GET_CHAT_STATE_COMMAND,
+type APIConfig = {
+  serverUrl?: string | undefined;
+  isSelfHosted: boolean;
+  isTelemetryEnabled: boolean;
+};
+
+export class ChatApi {
```

Comment:
this is  now a class that is inited [here](https://github.com/codota/tabnine-vscode/pull/1397/files#diff-c33546e0b57fc3ecb21320676107561d142900bbc9aa67ba0b18808cfd0b4e9fR120)
and [here](https://github.com/codota/tabnine-vscode/pull/1397/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R156)